README.md

```diff
@@ -487,6 +487,34 @@ crud.truncate('customers', {timeout = 2})
 ...
 ```
 
+### Len
+
+```lua
+-- Calculates the number of tuples in the space
+local result, err = crud.len(space_name, opts)
+```
+
+where:
+
+* `space_name` (`string`) - name of the space
```

Comment:
`space_name` can be a numerical id too. Please fix the description for argument and types here and in code.